py/selenium/webdriver/ie/webdriver.py

```diff
@@ -78,9 +83,6 @@ def __init__(self, executable_path='IEDriverServer.exe', capabilities=None,
 
         self.iedriver.start()
 
-        if capabilities is None:
```

Comment:
Removed this set of lines since the prior lines set capabilities to something. Although its interesting though that without any options or capabilites set, you're passing an empty dict to `RemoteWebDriver`.